docs/modules/usage/llms/google-llms.md

```diff
@@ -1,28 +1,43 @@
 # Google Gemini/Vertex LLM
 
-## Completion
-
 OpenHands uses LiteLLM for completion calls. The following resources are relevant for using OpenHands with Google's LLMs:
 
 - [Gemini - Google AI Studio](https://docs.litellm.ai/docs/providers/gemini)
 - [VertexAI - Google Cloud Platform](https://docs.litellm.ai/docs/providers/vertex)
 
-### Gemini - Google AI Studio Configs
+## Gemini - Google AI Studio Configs
 
-To use Gemini through Google AI Studio when running the OpenHands Docker image, you'll need to set the following environment variables using `-e`:
+To use Gemini through Google AI Studio when running OpenHands, you'll need to set the following environment variable
+using `-e` in the [docker run command](/modules/usage/getting-started#installation):
 
 ```
 GEMINI_API_KEY="<your-google-api-key>"
-LLM_MODEL="gemini/gemini-1.5-pro"
 ```
 
-### Vertex AI - Google Cloud Platform Configs
+Example:
+```bash
+docker run -it --pull=always \
+    -e GEMINI_API_KEY="AIzaSyA-1234567890abcdefGHIJKLmnopQRStuvWXYZ"
```

Comment:
This looks frighteningly like a real key. I'm almost having a heart attack here, it would not be the first real key that made it to this repo! Can we please use something like "AIzaSya-..." instead?

Reply:
haha. Will change. I just asked GPT "What a typical API key looks like for Gemini"

Reply:
Removed as it is not needed